spec/runtime_request_spec.rb

```diff
@@ -270,6 +270,46 @@
         expect(subject.path_parameters).to eq({})
       end
     end
+
+    context 'with kebab-case path params' do
+      let(:definition) do
+        OpenapiFirst.parse({
+                             'openapi' => '3.1.0',
+                             'paths' => {
+                               '/stuff/{ke-bab}/{under_score}' => {
```

Comment:
I'd add a kebab path here as well:
```suggestion
                               '/ke-bab/{ke-bab}/{under_score}' => {
```
and then on line 305 too :)